tests/tasks/setup_test.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Set platform/version specific variables
+  include_role:
+    name: linux-system-roles.vpn
+    tasks_from: set_vars
+    public: true
+
+- name: Change inventory_hostname
+  set_fact:
+    inventory_hostname={{ main_hostname }}
+
+- name: show current inventory hostname
+  debug:
+    msg: inv hostname {{ inventory_hostname }}
+
+- name: Create nss directory for testing
+  file:
+    path: "{{ __vpn_nss_location }}"
+    state: directory
```

Comment:
I think you have to add a `mode` here e.g. `mode: 0600` or you will get flagged by ansible-lint (unless that item is already suppressed)